TweetMate/src/main/java/com/ybsystem/tweetmate/adapters/array/DrawerArrayAdapter.java

```diff
@@ -0,0 +1,174 @@
+package com.ybsystem.tweetmate.adapters.array;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.ybsystem.tweetmate.R;
+import com.ybsystem.tweetmate.activities.SearchActivity;
+import com.ybsystem.tweetmate.activities.TimelineActivity;
+import com.ybsystem.tweetmate.activities.UserListActivity;
+import com.ybsystem.tweetmate.activities.preference.SettingActivity;
+import com.ybsystem.tweetmate.application.TweetMateApp;
+import com.ybsystem.tweetmate.models.enums.ColumnType;
+import com.ybsystem.tweetmate.databases.PrefAppearance;
+import com.ybsystem.tweetmate.utils.CalcUtils;
+import com.ybsystem.tweetmate.utils.DialogUtils;
+
+import static com.ybsystem.tweetmate.models.enums.ColumnType.*;
+import static com.ybsystem.tweetmate.resources.ResString.*;
+
+public class DrawerArrayAdapter extends ArrayAdapter<Integer> {
+
+    public DrawerArrayAdapter(Context context) {
+        super(context, android.R.layout.simple_list_item_1);
+    }
+
+    @Override
+    public View getView(int position, View convertView, ViewGroup parent) {
+        // Inflate
+        View cv = convertView;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "cv". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1854)